README.md

```diff
@@ -1,31 +1,40 @@
 # React Starter Kit
 
-To install the dependencies:
+## Up & Running
+• Install dependencies:
 
-```
-npm install
-```
+`npm install` or `yarn`
 
-To fire up a development server:
+• Fire up a development server:
 
-```
-npm start
+```js
+npm run dev
```

Comment:
This is great! `dev` vs. `prod` is very important to understand. Also are sourcemaps being used for clean breakpoints in DevTools? If not, it's pretty easy to understand that `_this2` is the this needed in most cases